yt/yt/server/lib/misc/job_reporter.cpp

```diff
@@ -74,7 +75,35 @@ class TJobRowlet
 
     size_t EstimateSize() const override
     {
-        return Report_.EstimateSize();
+        return ::NYT::EstimateSizes(
+            Report_.OperationId().Underlying(),
+            Report_.JobId().Underlying(),
+            Report_.State(),
+            Report_.StartTime(),
+            Report_.FinishTime(),
+            TInstant::Now().MicroSeconds(), // UpdateTime.
+            Report_.Address(),
+            Report_.StderrSize(),
+            Report_.HasCompetitors(),
+            Report_.HasProbingCompetitors(),
+            Report_.TaskName(),
+            Report_.TreeId(),
+            Report_.MonitoringDescriptor(),
+            Report_.Type(),
+            Report_.Error(),
+            Report_.InterruptionInfo(),
+            Report_.Statistics(),
+            Report_.Events(),
+            Report_.Spec().has_value(),
+            i64{0}, // FailContextSize.
```

Comment:
I'd suggest `/* failContextSize */ i64{0}`.

Reply:
Afaik we use `/*smth*/` without spaces now? Otherwise, sure.